server/server/src/sync/sync_api_v5.rs

```diff
@@ -85,9 +112,8 @@ impl SyncApiV5 {
     // Should only be called on initial sync or when re-initializing an existing data file.
     pub async fn post_initialise(&self) -> Result<RemoteSyncBatchV5, SyncConnectionError> {
         let url = self.server_url.join("/sync/v5/initialise")?;
-
         // Server rejects initialization request if no `content-length` header included.
-        let mut headers = HeaderMap::new();
+        let mut headers = self.headers.clone();
```

Comment:
@mark-prins  did you try posting an empty body here? I wonder if that would fix the need to add a content_length header manually?


Possibly something like this?
```
.post(url)
.body(Body::from(""))
```
Not sure if it would work but I think it might.

Officially the content length header isn't required if there is no body.